catalog-rest-service/src/main/resources/ui/src/components/common/entityPageInfo/EntityPageInfo.tsx

```diff
@@ -81,47 +96,90 @@ const EntityPageInfo = ({
           </span>
         ))}
       </div>
-      <div className="tw-flex tw-flex-wrap tw-pt-1">
-        {(tags.length > 0 || tier) && (
-          <i className="fas fa-tags tw-px-1 tw-mt-2 tw-text-grey-muted" />
-        )}
-        {tier && (
-          <Tags className="tw-bg-gray-200" tag={`#${tier.split('.')[1]}`} />
-        )}
-        {tags.length > 0 && (
+      <div className="tw-flex tw-flex-wrap tw-pt-1 tw-group">
+        {(!isEditable || !isTagEditable) && (
           <>
-            {tags.slice(0, LIST_SIZE).map((tag, index) => (
-              <Tags
-                className="tw-bg-gray-200"
-                key={index}
-                tag={`#${tag.tagFQN}`}
-              />
-            ))}
+            {(tags.length > 0 || tier) && (
+              <i className="fas fa-tags tw-px-1 tw-mt-2 tw-text-grey-muted" />
+            )}
+            {tier && (
+              <Tags className="tw-bg-gray-200" tag={`#${tier.split('.')[1]}`} />
+            )}
+            {tags.length > 0 && (
+              <>
+                {tags.slice(0, LIST_SIZE).map((tag, index) => (
+                  <Tags
+                    className="tw-bg-gray-200"
+                    key={index}
+                    tag={`#${tag.tagFQN}`}
+                  />
+                ))}
 
-            {tags.slice(LIST_SIZE).length > 0 && (
-              <PopOver
-                className="tw-py-1"
-                html={
-                  <>
-                    {tags.slice(LIST_SIZE).map((tag, index) => (
-                      <Tags
-                        className="tw-bg-gray-200 tw-px-2"
-                        key={index}
-                        tag={`#${tag.tagFQN}`}
-                      />
-                    ))}
-                  </>
-                }
-                position="bottom"
-                theme="light"
-                trigger="click">
-                <span className="tw-cursor-pointer tw-text-xs link-text">
-                  View more
-                </span>
-              </PopOver>
+                {tags.slice(LIST_SIZE).length > 0 && (
+                  <PopOver
+                    html={
+                      <>
+                        {tags.slice(LIST_SIZE).map((tag, index) => (
+                          <Tags
+                            className="tw-bg-gray-200 tw-px-2"
+                            key={index}
+                            tag={`#${tag.tagFQN}`}
+                          />
+                        ))}
+                      </>
+                    }
+                    position="bottom"
+                    theme="light"
+                    trigger="click">
+                    <span className="tw-cursor-pointer tw-text-xs link-text v-align-sub tw--ml-1">
```

Comment:
`link-text` adds underline to `...` and we do not have that anywhere else, we have many either of the behaviour consistent

Reply:
we do the same thing for column level joins as well.